changelogs/head.asciidoc

```diff
@@ -17,3 +17,4 @@ https://github.com/elastic/apm-server/compare/8.9\...main[View commits]
 
 [float]
 ==== Added
+- Add a transaction to the source map periodic refresh action. {pull}11116[11116]
```

Comment:
Would be great to mention this is for instrumentation only.

Reply:
How about now?